.buildkite/pipeline.yml

```diff
@@ -6,4 +6,15 @@ steps:
     label: ":xcode_simulator: Unit Test Pass"
   - wait
   - command: "./scripts/run-ui-tests.sh MuxPlayerSwiftExample"
-    label: ":xcode_simulator: Baseline Unit Test Pass"
+    label: ":xcode_simulator: UI Test Pass"
+  - wait
+  - command: "./scripts/create-example-application-archive.sh MuxPlayerSwiftExample"
+    label: ":xcode: Export Test Application and Test Runner Archives"
+    artifact_paths:
+      - "./Examples/MuxPlayerSwiftExample/MuxPlayerSwiftExample.ipa"
+      - "./Examples/MuxPlayerSwiftExample/Packaging.log"
```

Comment:
Along with the example `.ipa` package, upload Xcode logs and build summaries to help triage SL test failures.